src/librustdoc/html/static/settings.js

```diff
@@ -14,10 +14,24 @@
         }
     }
 
+    function handleKey(ev) {
+        // Don't interfere with browser shortcuts
+        if (ev.ctrlKey || ev.altKey || ev.metaKey) {
+            return;
+        }
+        switch (getVirtualKey(ev)) {
+            case "Enter":
+            case "Return":
+            case "Space":
+                ev.target.checked = !ev.target.checked;
+                break;
+        }
+    }
+
     function setEvents() {
         var elems = {
-            toggles: document.getElementsByClassName("slider"),
-            selects: document.getElementsByClassName("select-wrapper")
+            toggles: Array.prototype.slice.call(document.getElementsByClassName("slider")),
+            selects: Array.prototype.slice.call(document.getElementsByClassName("select-wrapper")),
```

Comment:
What does this do?

Reply:
A very "funny" bug. Basically, it's only possible to view it on chrome. `document.getElementsByClassName` returns an iterator, but every time you try to access a value, on chrome it goes through all the elements and then returns the one you asked, which made rustdoc pages very slow to render on chrome (and which is why I created the `onEachLazy` function).

In this case, it seems like they wanted to do it this way, but I guess using `onEachLazy` would make more sense?

Reply:
Ok, definitely better with `onEachLazy`. Sending the update soon.